chalice/deploy/deployer.py

```diff
@@ -59,6 +59,7 @@ def validate_configuration(config):
     """
     routes = config.chalice_app.routes
     validate_routes(routes)
+    validate_route_content_types(routes, config.chalice_app.api.binary_types)
```

Comment:
I am not sure about this either.  I went with public for mine since 2/3 were public.